generators/firechip/src/main/scala/Targets.scala

```diff
@@ -101,3 +101,15 @@ class FireSimTraceGenModuleImp(outer: FireSimTraceGen) extends BaseSubsystemModu
 
 // Supernoded-ness comes from setting p(NumNodes) (see DefaultFiresimHarness) to something > 1
 class FireSimSupernode(implicit p: Parameters) extends DefaultFireSimHarness(() => new FireSimDUT)
+
+// Verilog blackbox integration demo
```

Comment:
In the future it would be good to have a more substantial demo for this. Maybe we can have a ugrad black-box a RoCC accelerator.

Reply:
+1

Reply:
This is why we need to keep teaching undergrads Verilog in 151 :)